DeDriftAndResample/DeDriftAndResamplePipeline.sh

```diff
@@ -870,6 +914,26 @@ main()
 	    reapply_mr_fix_cmd=("${HCPPIPEDIR}/ICAFIX/ReApplyFixMultiRunPipeline.sh" --path="${StudyFolder}" --subject="${Subject}" --fmri-names="${mrFIXNames[$i]}" --concat-fmri-name="${mrFIXConcatNames[$i]}" --high-pass="${HighPass}" --reg-name="${ConcatRegName}" --matlab-run-mode="${MatlabRunMode}" --motion-regression="${MotionRegression}")
 	    log_Msg "reapply_mr_fix_cmd: ${reapply_mr_fix_cmd[*]}"
 	    "${reapply_mr_fix_cmd[@]}"
+	    
+	    extract_cmd=("${HCPPIPEDIR}/DeDriftAndResample/scripts/ExtractFromMRFIXConcat.sh"
```

Comment:
So, you decided to situate this Extract script in `DeDriftAndResample`, rather than `global`?

Reply:
It will be relocated to global/scripts.